tests/test_callback_executor.py

```diff
@@ -38,5 +39,5 @@ def test_callback_executor(self, mock_popen):
         self.assertIsNone(ce.call(callback))
 
         mock_popen.side_effect = [Mock()]
-        self.assertIsNone(ce.call(['test.sh', 'on_reload', 'replica', 'foo']))
+        self.assertIsNone(ce.call(['test.sh', 'on_reload', PostgresqlRole.REPLICA, 'foo']))
```

Comment:
Actually, here we need to check that callback is called with 'replica' as a string.